src/arch/riscv/kernel/boot.c

```diff
@@ -25,12 +25,12 @@ extern char ki_boot_end[1];
 extern char ki_end[1];
 
 #ifdef ENABLE_SMP_SUPPORT
-BOOT_DATA static volatile word_t node_boot_lock = 0;
+BOOT_BSS static volatile word_t node_boot_lock;
```

Comment:
Just to make sure our bootloaders clear the BSS.

Reply:
They all do.